apps/src/templates/instructions.html.ejs

```diff
@@ -5,10 +5,10 @@
   <div class='instructions-markdown<% if (locals.markdownClassicMargins) { -%> classic-margins<% } -%>'><%- locals.renderedMarkdown %></div>
 <% } else { -%>
   <% if (locals.instructions) { -%>
-    <p><%= locals.instructions %></p>
```

Comment:
Are you saying this is a downside because translators could put in html that we may not necessarily want?